### PR TITLE
fix: fix content manager port and error handling

### PR DESCRIPTION
**Pull request by Cafe137**

### CODE REVIEW

**Review by AuHau — APPROVED** [review]
LGTM! Thanks for the fixes, I am approving, but would like to @luissanchez0305 also review it, so please wait for the merge till his approval ;-)

**Review by luissanchez0305 — APPROVED** [review]

**Pull request merged**
